cmd/admin-handlers.go

```diff
@@ -256,47 +256,69 @@ func (adminAPI adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *htt
 	}
 	storage := objLayer.StorageInfo()
 
-	// Build list of enabled ARNs queues
-	var arns []string
-	for queueArn := range globalEventNotifier.GetAllExternalTargets() {
-		arns = append(arns, queueArn)
-	}
+	// Web service response
+	resp := make(map[string]*ServerInfo)
 
-	// Fetch uptimes from all peers. This may fail to due to lack
-	// of read-quorum availability.
-	uptime, err := getPeerUptimes(globalAdminPeers)
-	if err != nil {
-		writeErrorResponse(w, toAPIErrorCode(err), r.URL)
-		errorIf(err, "Unable to get uptime from majority of servers.")
-		return
-	}
+	// Gather server information for all nodes
+	for _, peer := range globalAdminPeers {
 
```

Comment:
ServerInfo should have a field where it has the peer name or address associated with it.